src/components/ListItem.jsx

```diff
@@ -58,6 +64,8 @@ export function ListItem({
 				/>
 				{name}
 			</label>
+			&nbsp;
+			<button onClick={handleDelete}>Delete</button>
 		</li>
```

Comment:
`thought:` it might be kind of neat to use a semantic trash icon in place of `delete` https://semantic-ui.com/elements/icon.html#:~:text=Icons%20serve%20a%20very%20similar,for%20its%20standard%20icon%20set.

Reply:
Thanks for the ideas. We can work on the design part for the next two weeks. :+1:

Reply:
Ooo that would look nice. Looking forward to adding that next week if everyone agrees :)